internal/core/command/controller/messaging/router.go

```diff
@@ -13,7 +13,7 @@ import (
 // MessagingRouter defines interface for Command Service to know
 // where to route the receiving device command response.
 type MessagingRouter interface {
-	// ResponseTopic returns the responseTopicPrefix by requestId, and a boolean value
+	// DeviceResponseTopic returns the responseTopicPrefix by requestId, and a boolean value
```

Comment:
I don't think you meant to change this one.
```suggestion
	// ResponseTopic returns the responseTopicPrefix by requestId, and a boolean value
```